hive_agent/llm_settings.py

```diff
@@ -22,18 +22,18 @@ def init_llm_settings(config):
     model = config.get("model", "model", "gpt-3.5-turbo")
     if "gpt" in model:
         Settings.llm = OpenAI(model=model)
-        logging.info(f"Model selected is openai")
+        logging.info(f"OpenAI model selected")
     elif "claude" in model:
         Settings.llm = Anthropic(model=model, api_key=os.getenv("ANTHROPIC_API_KEY"))
-        logging.info(f"Model selected is Claude")
+        logging.info(f"Claude model selected")
     elif "llama" in model:
-        Settings.llm = Ollama(model=model, api_key=os.getenv("LLAMA_API_KEY"))
-        logging.info(f"Model selected is Llama")
+        Settings.llm = Ollama(model="llama3", request_timeout=config.get("timeout", "ollama", 240))
```

Comment:
you would have to make a new variable and apply it to all the models, default should be 30